server/sonar-web/src/main/js/app/utils/getCurrentUserFromStore.js

```diff
@@ -22,9 +22,7 @@ import { getCurrentUser } from '../../store/rootReducer';
 
 // TODO remove my usages
 const getCurrentUserFromStore = () => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)